demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -51,7 +55,7 @@ def is_valid(self):
             return self._is_valid
 
         is_existing_image = False
-        self.validate_size(allow_empty_image_file=True)
+        self.validate_size(allow_empty_image_file=True, should_validate_dimensions=True)
```

Comment:
when do we not validate dimensions?

Reply:
We don't care about dimensions when validating AuthorImage, only in Integration image.

Reply:
Please add a comment mentioning it